docs/elasticsearch.md

```diff
@@ -14,195 +14,436 @@ sidebar_label: Elasticsearch
 
-   You must configure a StorageClass to provision cStor volume on given cStor pool. StorageClass is the interface through which most of the OpenEBS storage policies are defined. In this solution we are using a StorageClass to consume the cStor Pool which is created using external disks attached on the Nodes.  Since Elasticsearch is a StatefulSet, it requires only single storage replica. So cStor volume `replicaCount` is >=1. Sample YAML named **openebs-sc-disk.yaml**to consume cStor pool with cStor volume replica count as 1 is provided in the configuration details below.
+- `openebs-device` - Using this option, it will create Kubernetes Local PVs using the block devices attached to the node. Select this option when you want to dedicate a complete block device on a node to a Elasticsearch application pod. You can customize which devices will be discovered and managed by OpenEBS using the instructions [here](/docs/next/uglocalpv-device.html#optional-block-device-tagging). 
 
-5. **Launch and test Elasticsearch**
+The Storage Class `openebs-device` has been chosen to deploy Elasticsearch in the Kubernetes cluster.
 
-   Use latest Elasticsearch chart with helm to deploy Elasticsearch in your cluster using the following command. In the following command, it will create PVC with 30G size.
+**Note:** Ensure that you have a disk with the required capacity added to the corresponding nodes.
 
-   ```
-   helm install --name es-test --set volumeClaimTemplate.storageClassName=openebs-cstor-disk elastic/Elasticsearch --version 6.6.0-alpha1
-   ```
+### Installing KUDO Operator
 
-   For more information on installation, see Elasticsearch [documentation](https://github.com/elastic/helm-charts/tree/master/elasticsearch).
+In this section, we will install the KUDO operator. We will later deploy the latest available version of Elasticsearch using KUDO. 
 
-<br>
+Use the latest stable version of KUDO CLI. The latest version of KUDO can be checked from [here](https://github.com/kudobuilder/kudo/releases).
 
-<hr>
+#### Verify if Cert-manager is installed
 
-<br>
+For installing KUDO operator, the Cert-manager must be already installed in your cluster. If not, install the Cert-manager. The instruction can be found from [here](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests). Since our K8s version is v1.18.12, we have installed Cert-manager using the following command.
 
-## Reference at [openebs.ci](https://openebs.ci/)
+```bash
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+```
 
-<br>
+```bash
+kubectl get pods --namespace cert-manager
+```
+```
 
```

Comment:
Can we remove this extra blank line at the beginning of all outputs? We can do the same for all outputs? In the discussion, I was mentioning to add the space between command and output if both are representing in one code block,. Now you have separated command and output in a different code block, we don't need extra space. Also for all output, we can mention that the following is the sample output of the above command.